src/NextGen/DonationForm/resources/app/form/MultiStepForm/hooks/useNextStep.ts

```diff
@@ -0,0 +1,19 @@
+import {useDonationFormMultiStepStateDispatch} from '@givewp/forms/app/form/MultiStepForm/store';
+import {setCurrentStep} from '@givewp/forms/app/form/MultiStepForm/store/reducer';
+import {useDonationFormStateDispatch} from '@givewp/forms/app/store';
+import {setFormDefaultValues} from '@givewp/forms/app/store/reducer';
+import {FieldValues} from 'react-hook-form';
+
+/**
+ * @unreleased
+ */
+export default function useNextStep() {
+    const dispatchForm = useDonationFormStateDispatch();
+    const dispatchMultiStep = useDonationFormMultiStepStateDispatch();
+
+    return (currentStep: number, formValues: FieldValues) => {
+        dispatchForm(setFormDefaultValues(formValues));
+
+        dispatchMultiStep(setCurrentStep(currentStep + 1));
+    };
```

Comment:
Let's use `useCallback` here to memoize the returned function.

Reply:
Resolved 07066eb2